src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,373 @@
+### Verifying Your Setup
 
-### Serverless
+Great! Now that you’ve completed setting up the SDK, maybe you want to quickly test out how Sentry works. Start by capturing an exception:
 
-* [AWS Lambda]({% link _documentation/platforms/python/aws_lambda.md %})
-* [Generic Serverless]({% link _documentation/platforms/python/serverless.md %})
+```python
+sentry_sdk.capture_exception(Exception("This is an example of an error message."))
+```
 
-### Other Integrations
+Once the exception is captured, you'll see the error in your Sentry dashboard.
 
-* [Logging]({% link _documentation/platforms/python/logging.md %})
-* [GNU Backtrace]({% link _documentation/platforms/python/gnu_backtrace.md %})
-* [Default integrations]({% link _documentation/platforms/python/default-integrations.md %})
+## Capturing Errors
 
-{% capture __alert_content -%}
-Looking for [the legacy Python SDK "Raven"?]({%- link _documentation/clients/python/index.md -%})
-{%- endcapture -%}
-{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-%}
+In Python you can either capture a caught exception or the one currently held in `sys.exc_info()` by not passing an argument:
+
+```python
+from sentry_sdk import capture_exception
+
+try:
+    a_potentially_failing_function()
+except Exception as e:
+    # Alternatively the argument can be omitted
+    capture_exception(e)
+```
+
+## Releases
+
+A release is a version of your code that you deploy to an environment. When you give Sentry information about your releases, you unlock many new features:
+
+- Determine the issue and regressions introduced in a new release
+- Predict which commit caused an issue and who is likely responsible
+- Resolve issues by including the issue number in your commit message
+- Receive email notifications when your code gets deployed
+
+After configuring your SDK, setting up releases is a 2-step process:
+
+1. [Create Release and Associate Commits]({%- link _documentation/workflow/releases.md -%}#create-release)
+2. [Tell Sentry When You Deploy a Release]({%- link _documentation/workflow/releases.md -%}#create-deploy)
+
+For more information, see [Releases Are Better With Commits](https://blog.sentry.io/2017/05/01/release-commits.html).
+
+## Context
+
+Sentry supports additional context with events. Often this context is shared among any issue captured in its lifecycle, and includes the following components:
+
+**Structured Contexts**
+
+: Structured contexts are typically set automatically.
+
+[**User**](#capturing-the-user)
+
+: Information about the current actor
+
+[**Tags**](#tagging-events)
+
+: Key/value pairs which generate breakdown charts and search filters
+
+[**Level**](#setting-the-level)
+
+: An event's severity
+
+[**Fingerprint**](#setting-the-fingerprint)
+
+: A value used for grouping events into issues
+
+[**Unstructured Extra Data**](#extra-context)
+
+: Arbitrary unstructured data which the Sentry SDK stores with an event sample
+
+### Setting Context
```

Comment:
@untitaker will you double check my work for this section "Setting Context" and also the section for "Extra Context?" Specifically, am I explaining the difference between .set_context() and .set_extra() correctly?

Reply:
`set-context` has reserved keys that have a special meaning while `set-extra` doesn't. One could say `set-extra` is for data that doesn't fit into our existing event schema.

`set-extra` *does* accept objects as second argument though.